oembed/tests.py

```diff
@@ -1,32 +1,37 @@
+from __future__ import unicode_literals
+
 from django.test import TestCase
 from oembed.core import replace
 
 class OEmbedTests(TestCase):
-    noembed = ur"This is text that should not match any regex."
-    end = ur"There is this great video at %s"
-    start = ur"%s is a video that I like."
-    middle = ur"There is a movie here: %s and I really like it."
-    trailing_comma = ur"This is great %s, but it might not work."
-    trailing_period = ur"I like this video, located at %s."
-    
-    loc = u"http://www.viddler.com/explore/SYSTM/videos/49/"
```

Comment:
Viddler's oembed api is no longer available, so I've updated the tests to look at Flickr's, seen here: https://www.flickr.com/services/oembed/?url=https://www.flickr.com/photos/ian_ruotsala/39088280250/in/feed